src/Ledger/GovernanceActions.lagda

```diff
@@ -174,7 +187,9 @@ instance
 The data type \Vote represents the different voting options: \yes,
 \no, or \abstain. To cast a \Vote, it needs to be packaged together
 with further information, such as who votes and for which governance
```

Comment:
```suggestion
\no, or \abstain. For a \Vote to be cast, it must be packaged together
with further information, such as who votes and for which governance
```

Reply:
(this makes the sentence easier to read by making it more obvious from the start that the subject is the vote, not the person casting it)